imports/plugins/core/tags/server/no-meteor/queries/productsByTagId.js

```diff
@@ -50,16 +50,32 @@ export default async function productsByTagId(context, params) {
     }
   };
 
-  // Sort the results by "__order"
+  // Projection: Add a featuredPosition by order
+  const projection = {
+    $project: {
+      _id: 1,
+      position: 1,
+      title: 1,
+      sortPosition: {
+        $cond: {
+          if: { $lt: ["$position", 0] },
+          then: { $add: [{ $abs: "$position" }, positions.length] },
```

Comment:
`positions` will be null if a Tag doesn't have any `featuredProductIds` yet, and `positions.length` will fail with: ` "Cannot read property 'length' of undefined",`